lib/msal-browser/docs/testing.md

```diff
@@ -0,0 +1,138 @@
+# Testing
+
+## The loadExternalTokens() API
+
+MSAL Browser starting version 2.17.0 has added the `loadExternalTokens()` API, which allows the loading of id tokens and access tokens to the MSAL cache, which can then be read out using `acquireTokenSilent()`. 
+
+**Note: This is an advanced feature that is intended for testing purposes in the browser environment only. Loading tokens to your application's cache may cause your app to break.**
+
+The `loadExternalTokens()` API can be accessed by calling `getTokenCache()` on MSAL Browser's `PublicClientApplication` instance. 
+
+```js
+const msalTokenCache = myMSALObj.getTokenCache();
+msalTokenCache.loadExternalTokens(silentRequest, serverResponse, loadTokenOptions);
+```
+
+`loadExternalTokens()` takes in a request of type `SilentRequest`, a response of type `ServerAuthenticationROPCResponse`, and options of type `LoadTokenOptions`.
+
+See the type definitions for each, which can be imported from `@azure/msal-browser`:
+
+- [`SilentRequest`](https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_browser.html#silentrequest)
+- [`ServerAuthorizationROPCResponse`](https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_common.html#serverauthorizationropcresponse)
```

Comment:
Are we changing this name? As @tnorling suggested in a different comment.

Reply:
Yes! Changed